src/moduler/feilmelding/IkkeNiva4.tsx

```diff
@@ -0,0 +1,34 @@
+import { useSelector } from 'react-redux';
+import { selectNivaa4 } from '../tilgang/tilgang-selector';
+import classNames from 'classnames';
+import React from 'react';
+
+import styles from './Feilmelding.module.less';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+
+const Nivaa4Feilmelding = () => {
+    const niva4 = useSelector(selectNivaa4);
+
+    if (niva4) {
+        return null;
+    }
+
+    return (
+        <div className={classNames(styles.feilmelding)}>
```

Comment:
Her blir vel dette det samme som det under her?
```suggestion
        <div className={styles.feilmelding}>
```